.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+  - 1.4.1
```

Comment:
@juliusv, @brian-brazil - happy to change it. one advantage on having the versions in the travis file is it will compile the project against multiple versions of go, [travis go](http://docs.travis-ci.com/user/languages/go/), and will also document which versions of go the library is compatible with.  I unintentionally had an older version of go installed (1.3) and the project wouldn't compile on it, which is what motivated me to add travis to this project.